docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,111 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+Validation and Verification Criteria
+------------------------------------
 
-   all of this
+Obejective
+++++++++++
+External
+ - Model results should be checked against local TB epidemiology
+    - e.g. rate of decline in burden of disease should be compared
+      with historical evidence.
+ - Compare our results (e.g. ICERs) to similar models or empirical
+   assessments where possible
+Internal
+ - Test hypotheses without develop a full simulation. 
+ - Calibrate simulation baseline against GBD 2017 results
+    - Sim outputs mean should perfectly match GBD results.
+    - Sim outputs uncertainty should be wider than GBD results,
+      because both stochastic and parametric uncertainty are included.
+
+V&V strategy
+++++++++++++
+Model Validation
+ - Check the logical structure and input data for concept model,
+   make sure
+    - the theories and assumptions underlying the conceptual model are correct.
+    - the data to build, evaluate, and test model are correct.
```

Comment:
Also that the model is sufficiently complex to capture the necessary dynamics to produce a valid result